examples/graphene_apps/python_worker/avalon_worker/workload_processor.py

```diff
@@ -0,0 +1,127 @@
+        Returns :
+            status as boolean and outData dictionary.
+        """
+        wl_processor = self._create_workload_processor(workload_id)
+        if not wl_processor:
+            logger.error("unable to create work load processor")
+            return False, None
+
+        result, out_msg_bytes = wl_processor.execute(in_data_array)
+        out_data = self._create_response_out_data(out_msg_bytes)
+        return result, out_data
+
+# -------------------------------------------------------------------------
+
+    def _create_workload_processor(self, workload_id):
+        """
+        Executes workloads based on workload id.
+
+        Parameters :
+            workload_id:  Workload id to execute.
+            in_data_array: Input data array containing data in plain bytes.
+        Returns :
+            status as boolean and outData dictionary.
+        """
+        instance = None
+        try:
+            with open("workloads.json", "rb") as file:
+                workload_json = json.load(file)
+        except Exception as e:
+            logger.error("Work load processor creation failed" + str(e))
+            return None
+
+        if workload_id in workload_json:
+            if "module" not in workload_json[workload_id]:
+                err_msg = "Work load module not found in workloads.json"
+                logger.error(err_msg)
+                return None
+            if "class" not in workload_json[workload_id]:
+                err_msg = "Work load class not found in workloads.json"
+                logger.error(err_msg)
+                return None
+            # import module
+            try:
+                module_name = workload_json[workload_id]["module"]
+                class_name = workload_json[workload_id]["class"]
+                module = importlib.import_module(module_name)
+                class_ = getattr(module, class_name)
+                instance = class_()
+            except Exception as e:
+                logger.error("Work load processor creation failed: " + str(e))
+                return None
+        else:
+            err_msg = "{} workload id not found".format(workload_id)
+            logger.error(err_msg)
+            return None
+
+        return instance
+
+# -------------------------------------------------------------------------
+
+    def _create_response_out_data(self, out_data_plain_bytes):
+        """
+        Create output Work Order Data Format as per TC spec v1.1 section 6.5.
+
+        Parameters :
+            out_data_plain_bytes:  output data in plain bytes
+        Returns :
+            output Work Order Data Format as per TC spec v1.1 section 6.5.
+        """
+        out_data = dict()
+        out_data["data"] = out_data_plain_bytes
+        out_data["index"] = 0
+        # Data hash is optional. Ignore it for now.
+        # out_data["dataHash"] = ""
+        out_data["encryptedDataEncryptionKey"] = "null"
```

Comment:
Any purpose to put null within quotes here?

Reply:
This is needed as per EEA Trusted Compute spec v1.1 section 6 Work Order Data Formats